functions/extractIdFromParam.ts

```diff
@@ -0,0 +1,27 @@
+import type { FunctionContext, LoaderFunction } from "$live/types.ts";
```

Comment:
This function should be called something like productIdFromVTEXSlug, them we can add specific functions for specific plataforms. Also, we can depend on the params so we can compose everyting on the admin

```suggestion
import type { FunctionContext, LoaderFunction } from "$live/types.ts";
import type { RequestURLParam } from './requestToParam.ts'

export type ProductID = string;

export interface Props {
  /**
   * @default slug
   * @description Param name to extract from the Request URL
   */
  slug: RequestURLParam;
}

/**
 * @title Extract ID from request parameters
 * @description Set param to slug for routes of type /:slug
 */
const productIdFromVTEXSlug: LoaderFunction<
  Props,
  ProductID,
  FunctionContext
> = (req, ctx, { slug }) => {
  return slug.split('-').at(-1)
};

export default productIdFromVTEXSlug;
```